cli.js

```diff
@@ -112,28 +144,99 @@ async function verifyHerokuLogin() {
 
 require("yargs").usage(
   "$0 <name>",
-  "start the application server",
+  "Creates a new Bison application",
   function (yargs) {
     yargs.positional("name", {
       describe: "Creates a new Bison app with the specified name",
       type: "string",
     });
+
+    const options = generateQuestions();
+
+    options.forEach((option) => {
+      yargs.option(option.name, {
+        describe: option.description,
+        type: convertInquirerTypeToYarnType(option.type),
+      });
+    });
   },
   async function (yargs) {
     // Show the logo!
     console.log(Logo);
 
-    const { name } = yargs;
-    const questions = generateQuestions(name);
-    const answers = await inquirer.prompt(questions);
+    function parseObjectFromDotNotation2(obj) {
+      if (!obj) {
+        return {};
+      }
+
+      const isPlainObject = (obj) =>
+        !!obj && obj.constructor === {}.constructor;
+
+      const getNestedObject = (obj) =>
+        Object.entries(obj).reduce((result, [prop, val]) => {
+          prop.split(".").reduce((nestedResult, prop, propIndex, propArray) => {
+            const lastProp = propIndex === propArray.length - 1;
+            if (lastProp) {
+              nestedResult[prop] = isPlainObject(val)
+                ? getNestedObject(val)
+                : val;
+            } else {
+              nestedResult[prop] = nestedResult[prop] || {};
+            }
+            return nestedResult[prop];
+          }, result);
+          return result;
+        }, {});
+
+      return getNestedObject(obj);
+    }
+
+    async function fetchAnswers(answers = {}) {
+      const questions = generateQuestions(name);
+      const answerKeys = Object.keys(answers);
+
+      // filter out yargs keys from answers
+      const filteredAnswers = answerKeys.reduce((prev, key) => {
+        if (key.match(/(_|\$0)/)) {
+          return prev;
+        }
+
+        prev[key] = answers[key];
+        return prev;
+      }, Object.create(null));
+
+      // use defaults if acceptDefaults is true
+      const shouldUseDefaults = answerKeys.find((k) => k === "acceptDefaults");
```

Comment:
if the user passes `acceptDefaults` we merge any answers they explicitly added via commandline arguments